egs/aishell/ASR/pruned_transducer_stateless7/export2.py

```diff
@@ -100,27 +100,7 @@
     find_checkpoints,
     load_checkpoint,
 )
-from icefall.utils import str2bool
-
-
-def num_tokens(
-    token_table: k2.SymbolTable, disambig_pattern: str = re.compile(r"^#\d+$")
-) -> int:
-    """Return the number of tokens excluding those from
-    disambiguation symbols.
-
-    Caution:
-      0 is not a token ID so it is excluded from the return value.
-    """
-    symbols = token_table.symbols
-    ans = []
-    for s in symbols:
-        if not disambig_pattern.match(s):
-            ans.append(token_table[s])
-    num_tokens = len(ans)
-    if 0 in ans:
-        num_tokens -= 1
-    return num_tokens
+from icefall.utils import str2bool, num_tokens
```

Comment:
Please use `isort` to sort imports alphabetically.

Reply:
OK!